ui/src/components/Search/index.js

```diff
@@ -166,7 +166,7 @@ const Search = ({ isSearchOpen, setIsSearchOpen, value, setValue }) => {
                   variant="project"
                   image={project.avatar}
                   to={`/project/${project.id}`}
-                  category={project.categories[0].name}
+                  category={project.categories[0] ? project.categories[0].name : "None"}
```

Comment:
Gatsby's Babel config should support optional chaining as well as null-coalescing oprators if am not mistaken. This should work too:

```
category={project.categories?.[0]?.name ?? "None"}
```

Reply:
If the "categories" array doesn't exist for a project, it's going to break a lot of things elsewhere as well. It would be best to find out how the user added this category-less project, I will still agree to this change as the search bar is breaking for a lot of people because of it.

Reply:
Hi @KingZee categories are required when you create a project through the UI, but I am going to double check in the contracts whether that's the case there as well. The above linked project doesn't seem to exist anymore. 
In your PR - can you please replace "None" with just an empty string `""` (we don't want to show `None` on the card)

Reply:
Update: even if there are no categories, querying the subgraph would return an empty array for categories `[]` so we should be good there. I can check the code for other places where we didn't put `project.categories?.[0]?` check in place so it doesn't break. Unfortunately, I can't reproduce the original error :-/